docs/file_changes.md

```diff
@@ -130,7 +145,21 @@ is equivalent to having a `.dockerignore` file that looks like:
 
 The `only=` parameter accepts paths, not glob patterns.
 
-Again, these paths are evaluated relative to the `context` passed to `docker_build`; in the call above, the only directories included in the resulting image are `foo/src` and `foo/static-files`.
+Again, these paths are evaluated relative to the `context` passed to
+`docker_build`; in the call above, the only directories included in the
+resulting image are `foo/src` and `foo/static-files`.
+
+### only= and other build types
+
+Other build types don't have an `only=` parameter.
+
+In a `docker_build`, you're creating a docker build context to copy into the
+image.  The Docker build context is [an important
+concept](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#understand-build-context)
+for writing efficient Dockerfiles. The `only=` parameter lets you select
+specific directories in the build context.
+
+In other build types, you can specify multiply directories with `deps=`.
```

Comment:
I think we could get a bit more explicit about the difference, along the lines of:

For `docker_build`, `only` filters what files are actually available to the docker build within the build context. Other types of builds are simply able to access all files, and `deps=` is used to specify which paths trigger them.

Or, if you don't like that framing, in this last line: talking about "multiple" seems like it'd confuse more than clarify, and I'd swap out "directories" for "paths", since they can be files.

Reply:
i like that wording! done